charts/coredns/pod-delete/coredns-pod-delete.chartserviceversion.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: litmuchaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
+  name: coredns-pod-delete
+  version: 0.1.0
+  annotations:
+    categories: Kubernetes
+    vendor: CNCF
+    createdAt: 2019-12-27T10:28:08Z
+    support: https://slack.kubernetes.io/
+spec:
+  displayName: pod-delete
+  categoryDescription: |
+    Pod delete contains chaos to disrupt state of coredns resources. Experiments can inject random pod delete failures against specified application.
+    - Causes (forced/graceful) pod failure of coredns replicas of an application deployment.
+    - Tests deployment sanity (replica availability & uninterrupted service) and recovery workflows of the coredns pod.
```

Comment:
can we mention about verifying successful resolution of app services despite coredns pod failures

Reply:
done